src/TLuaInterpreter.cpp

```diff
@@ -13116,8 +13098,6 @@ void TLuaInterpreter::initLuaGlobals()
     lua_register( pGlobalLua, "roomLocked", TLuaInterpreter::roomLocked );
     lua_register( pGlobalLua, "setCustomEnvColor", TLuaInterpreter::setCustomEnvColor );
     lua_register( pGlobalLua, "getCustomEnvColorTable", TLuaInterpreter::getCustomEnvColorTable );
-    //lua_register( pGlobalLua, "setLevelColor", TLuaInterpreter::setLevelColor );
-    //lua_register( pGlobalLua, "getLevelColorTable", TLuaInterpreter::getLevelColorTable );
```

Comment:
I think these are intended to manipulate the colouration that is done for different z-axis offsets for drawing things in the 3D Mapper - currently the colour table is hard coded - but these do highlight that we could provide a means to customise them (the current choice __is__ a bit _meh_ IMHO!) so could you leave them and insert a `// TODO:` above them? :pray: